scripts/app_entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
the scripts folder is mixing two concerns:

1. this file as entrypoint for the RAGapp image (a local concern only for the image)
2. the `create_qdrant_collection.sh` is a shared script useful for all deployments

so I suggest:
1. move this script to the root (`/entrypoint.sh`) and add it inside of the `Dockerfile` (doesn't hurt to always use it - if `BASE_URL` is not specified, we don't need to patch). Once RAGapp is moved to its own folder (`/src/ragapp`), it will end up in (`/src/ragapp/entrypoint.sh`) 
2. move `/scripts/create_qdrant_collection.sh` to `/deployments/scripts/create_qdrant_collection.sh`

Reply:
That makes sense. Thanks for the suggestion!